README.md

```diff
@@ -45,6 +45,26 @@ Ex. If you are literally running build.sh locally..
 ```bash
 $ docker-machine create --driver virtualbox --virtualbox-memory "4096" dev
 ```
+
+## Connecting to Cassandra for Span Storage
+
+`zipkin-collector` and `zipkin-query` store and retrieve spans from Cassandra, using its native protocol. Specify a list of one or more Cassandra nodes listening on port 9042, via the comma-separated environment variable `CASSANDRA_CONTACT_POINTS`.
+
+ex. 
+```bash
+docker run -d -p 9411:9411 -p 9901:9901 --name="zipkin-query" -e "CASSANDRA_CONTACT_POINTS=node1,node2,node3" "itszero/zipkin-query:latest"
+docker run -d -p 9410:9410 -p 9900:9900 --name="zipkin-collector" -e "CASSANDRA_CONTACT_POINTS=node1,node2,node3" "itszero/zipkin-collector:latest"
+```
+
+Replace `node1,node2,node3` in above commands with your Cassandra nodes.
+
+If you are fine with running zipkin-query and zipkin-web on the same host you can still use a container link to connect those 2. eg:
```

Comment:
If we would have different config files for different back-end stores which use the same variables (eg `CONTACT_POINTS`) and we would allow specifying the configuration as a parameter we would be really independent.  Currently we are always referring the the Cassandra config.